internal/telemetry/event.go

```diff
@@ -164,7 +164,7 @@ func withAnonymousID() EventOpt {
 
 func WithDeploymentType(t string) EventOpt {
 	return func(event Event) {
-		event.Properties["deployment_type"] = t
+		event.Properties["deployment_type"] = strings.ToLower(t)
```

Comment:
I would day the commands should normalize it so `opts.DeploymentType` is always the correct value, since I see we don't guarantee it

Reply:
good point, will update!